hw/ip/csrng/doc/_index.md

```diff
@@ -129,6 +129,12 @@ The specific state information held in the instance is documented below.
 If the command was a `generate` command, the genbits data word will be returned to the requesting `cmd_stage` block.
 Finally, an `ack` response and status will be returned to the application interface once the command has been completely processed.
 
+A power spike feature exists where is a handshake exists between ENTROPY_SRC and CSRNG modules.
```

Comment:
Do the handshake signals appear in the `Hardware Interfaces` section? Without that, its hard to correlate those IOs with this feature description. Ditto comment on the entropy src doc below.

Reply:
After looking at other module specifications, I think it would be best to move these descriptions into a table that sits under the Hardware Interfaces section. This will make it look more consistent. Does this make sense @sriyerg ?

Reply:
Yes, that works, thanks!

Reply:
@sriyerg see if the tables look better, thanks.